mappings/net/minecraft/block/LeveledCauldronBlock.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5556 net/minecraft/block/LeveledCauldronBlock
+	FIELD field_27206 LEVEL Lnet/minecraft/class_2758;
+	FIELD field_27880 RAIN_PREDICATE Ljava/util/function/Predicate;
+	FIELD field_27881 SNOW_PREDICATE Ljava/util/function/Predicate;
+	FIELD field_27882 precipitationPredicate Ljava/util/function/Predicate;
+	METHOD <init> (Lnet/minecraft/class_4970$class_2251;Ljava/util/function/Predicate;Ljava/util/Map;)V
+		ARG 1 settings
+		ARG 2 precipitationPredicate
+		ARG 3 behaviorMap
+	METHOD method_31650 subtractWaterLevel (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
```

Comment:
`subtractFluidLevel` or `subtractLevel`?

Reply:
Fixed in 49d20674ee2ce67d420976d1efd48442dcd74db9